core/types/block.go

```diff
@@ -92,12 +92,12 @@ type Header struct {
 	// BlockSignature was added by EuclidV2 to make Extra empty and is ignored during hashing.
 	// This field is stored in db but not included in messages sent on the network wire protocol,
 	// or in RPC responses. See also `PrepareForNetwork` and `PrepareFromNetwork`.
-	BlockSignature []byte `json:"-" rlp:"optional"`
+	BlockSignature []byte `json:"-" rlp:"-"`
```

Comment:
This is just for the SDK? Because this can't be like this in mainnet or else loading blocks from memory will lead to issues.

Reply:
Correct, this PR cannot be merged.